File: acm/accepted/c/494.c
#include <stdio.h>
/*
	@JUDGE_ID:  20044WC 494 C "Siliconism - LFM"
*/
int main() {
	int i,j,k;
	char c;
	int state;

	while (1) {
		state = 0;
		k = 0;
		while (1) {
			c = fgetc(stdin);
			if (c == EOF) { return 0; };
			if (c == 10) { break; };
			if ((((c >= 'a') && (c <= 'z')) || ((c >= 'A') && (c <= 'Z'))) && (state == 0)) { state = 1; k ++ ; };
			if (!(((c >= 'a') && (c <= 'z')) || ((c >= 'A') && (c <= 'Z')))) { state = 0 ; } ;
		};
		printf("%d\n", k);
	};
};

File: acm/accepted/494.pas
program p494;
{
	@JUDGE_ID:  20044WC 494 PASCAL "DaP/DyP"
}
var
	state : byte;
	c : char;
	w : longint;
function alphabet(x : char) : boolean;
begin
	alphabet := false;
	if (x >= 'A') and (x <= 'Z') then alphabet := true;
	if (x >= 'a') and (x <= 'z') then alphabet := true;
end;

begin
	while not eof do begin
		state := 0;
		w := 0;
		while not eoln do begin
			read(c);
			if (state = 0) and (alphabet(c)) then begin
				inc(w);
				state := 1;
			end else if not alphabet(c) then begin
				state := 0;
			end;
		end;
		if not eof then writeln(w);
		readln;
	end;
end.
